package.json

```diff
@@ -39,5 +43,15 @@
     "karma-junit-reporter": "^1.2.0",
     "karma-phantomjs-launcher": "^1.0.2",
     "karma-requirejs": "^1.1.0"
-  }
+  },
+  "keywords": [
```

Comment:
Please synchronize these keywords with those in use on https://github.com/NASAWorldWind/WebWorldWind